appium/api-mocks/apis/google/config.ts

```diff
@@ -0,0 +1,44 @@
+import { KeyUtil, KeyInfoWithIdentityAndOptionalPp } from '../../../api-mocks/core/crypto/key';
+
+export type TestKeyInfo = {
+  title: string, passphrase: string, armored: string | null, longid: string | null
+};
+
+interface TestSecretsInterface {
+  ci_admin_token: string;
+  auth: { google: { email: string, password?: string, secret_2fa?: string }[], };
```

Comment:
I couldn't find usage for `ci_admin_token` and `auth`. Is this necessary?

Reply:
Probably we can completely remove `config.ts`, as `getKeyInfo` is used only for functionality migrated from browser tests and won't be needed for Appium tests.

Reply:
sounds good